cvat/apps/events/formatter.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2023 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import logging
+
+class CvatFormatter(logging.Formatter):
```

Comment:
class EventFormatter

Could you please write a comment about why we need it? It looks dummy.